pt-BR/mail/chrome/messenger/msgHdrViewOverlay.dtd

```diff
@@ -76,15 +76,15 @@
 <!ENTITY copyLinkCmd.label        "Copiar link">
 <!ENTITY copyLinkCmd.accesskey    "o">
 
-<!ENTITY CopyMessageId.label "Copy Message-ID">
-<!ENTITY OpenMessageForMsgId.label "Open Message For ID">
-<!ENTITY OpenBrowserWithMsgId.label "Open Browser With Message-ID">
+<!ENTITY CopyMessageId.label "Copiar Message-ID">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY CopyMessageId.label "Copiar ID da mensagem">